mlkem/indcpa.c

```diff
@@ -317,6 +323,14 @@ __contract__(
   }
 }
 
+/* Reference: `indcpa_keypair_derand()` in the reference implementation.
+ *            - We use x4-batched versions of `poly_getnoise` to leverage
+ *              batched x4-batched Keccak-f1600.
+ *            - We use a different implementation of `gen_matrix()` which
+ *              uses x4-batched Keccak-f1600 (see `mlk_gen_matrix()` above).
+ *            - We use a mulcache to speed up matrix-vector multiplication.
+ *            - We include buffer zeroization.
```

Comment:
Move zeroization to the list of major changes and don't list it for the individual functions

Reply:
I prefer to list them here too, but yes, it should be mentioned in the REFERENCe.md